.azure-pipelines/advanced.yml

```diff
@@ -11,8 +11,15 @@ schedules:
       - master
     always: true
 
-jobs:
-  # Any addition here should be reflected in the advanced-test and release pipelines.
-  # It is advised to declare all jobs here as templates to improve maintainability.
-  - template: templates/tests-suite.yml
-  - template: templates/installer-tests.yml
+stages:
+  - stage: Test_Package
+    jobs:
+      # Any addition here should be reflected in the advanced-test and release pipelines.
+      # It is advised to declare all jobs here as templates to improve maintainability.
+      - template: templates/jobs/standard-tests-jobs.yml
+      - template: templates/jobs/extended-tests-jobs.yml
+      - template: templates/jobs/packaging-jobs.yml
+  - stage: Deploy
```

Comment:
Initially because it was quite short to add it in each pipeline, and some other stages contents were also explicitly defined in the pipeline, so it was not adding a significant advantage. But with the refactoring it makes sense to align the approach on that matter (delegating actual stage definition in templates).

So done, also for the `Deploy` stage.